src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo: { completed, title, user } }) => {
+  const completedClass = completed ? ' TodoInfo--completed' : '';
+
+  return (
+    <article className={`TodoInfo${completedClass}`}>
+      <h2 className="TodoInfo__title">{title}</h2>
```

Comment:
it's no big problem with empty character in classes
but it will be more readability with spaces in code
```suggestion
  const completedClass = completed ? 'TodoInfo--completed' : '';

  return (
    <article className={`TodoInfo ${completedClass}`}>
      <h2 className="TodoInfo__title">{title}</h2>
```

Reply:
Thanks for the advice!)